internal/host/static/repository_host.go

```diff
@@ -73,14 +75,18 @@ func (r *Repository) CreateHost(ctx context.Context, scopeId string, h *Host, op
 
 	if err != nil {
 		if errors.IsUniqueError(err) {
-			return nil, fmt.Errorf("create: static host: in catalog: %s: name %s already exists: %w",
-				h.CatalogId, h.Name, errors.ErrNotUnique)
+			return nil, errors.New(errors.NotUnique,
+				errors.WithMsg(fmt.Sprintf("in catalog: %s: name %s already exists", h.CatalogId, h.Name)),
+				errors.WithCallerOp(false))
 		}
 		if errors.IsCheckConstraintError(err) || errors.IsNotNullError(err) {
-			return nil, fmt.Errorf("create: static host: in catalog: %s: %q: %w",
-				h.CatalogId, h.Address, ErrInvalidAddress)
+			return nil, errors.New(errors.NotNull,
```

Comment:
There are a few potential constraints that could be failing here, one of which would be the length of the address. (constraints address_must_be_more_than_2_characters and address_must_be_less_than_256_characters are the names of those constraints)  I wonder if we want to have different errors for the length validation checks and other validation checks (like not null)